src/app/progress/TrackedTriumphs.tsx

```diff
@@ -32,6 +33,25 @@ export function TrackedTriumphs({
     );
   }
 
+  // determine absolute path of record for sorting purpose
+  const record_path = (r: DestinyRecordDefinition) => {
```

Comment:
Match style with the rest of the code:

```suggestion
  const recordPath = (r: DestinyRecordDefinition) => {
```

Reply:
updated style